Lombiq.HelpfulLibraries/Libraries/Database/SchemaBuilderExtensions.cs

```diff
@@ -36,7 +36,14 @@ public static ISchemaBuilder AddDatabaseIndex<TTable>(this ISchemaBuilder schema
         {
             if (columnNames?.Any() != true)
             {
-                throw new ArgumentException($"{nameof(columnNames)} must contain at least one column name");
+                throw new ArgumentException("You must provide at least one column name.", nameof(columnNames));
+            }
+
+            if (columnNames.Any(string.IsNullOrWhiteSpace))
+            {
+                throw new ArgumentException(
+                    "The column names shouldn't be null, empty string or all whitespace.",

```

Comment:
I think it should be one of the following:

```suggestion
                    "The column names shouldn't be null, empty or all whitespace.",
```

Reply:
Or:

```suggestion
                    "The column names shouldn't be null, the empty string or all whitespace.",

```

Reply:
Or even:

```suggestion
                    "The column names should neither be null nor empty nor all whitespace.",

```

Reply:
I understand the first and third, but please explain what's the deal with the second. I don't get why "empty sting" should get a definite article.

Reply:
I guess an indefinite article would also be OK, i.e. "an empty string", but in my understanding there's only one empty string, or saying it differently, all empty strings are identical. But you can't use the phrase without any article: "The column names should not be empty string." won't work.

Reply:
Btw, please don't resolve conversations with open questions because GitHub won't expand them automatically and it's hard to find them when they are resolved 😉

Reply:
That was GitHub, not me. When you accept a suggestion it auto-resolves.
It's a bit annoying but I don't think it can be turned off.

Reply:
Oh, I see. Never mind then.